<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSetupDialog</class>
 <widget class="QDialog" name="DatabaseSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../universal/debtcollection/res.qrc">
    <normaloff>:/images/wheel.ico</normaloff>:/images/wheel.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>MySQL Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>User Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Host Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtHostName"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>175</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtUserName"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="cmdTestConnection">
         <property name="text">
          <string>Test Connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Other Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="txtSMTPServer"/>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QCheckBox" name="chkSMTPSSL">
         <property name="text">
          <string>SMTP SSL</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QCheckBox" name="chkSMTPAuth">
         <property name="text">
          <string>SMTP Authentication</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Tlm Server:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="txtSMTPPort"/>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="txtTLMPort"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>SMTP Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="txtTlmServer"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>SMTP Server:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QLineEdit" name="txtSMTPUserName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>TLM Port:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>SMTP User Name:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_9">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>SMTP Password:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLineEdit" name="txtSMTPPassword">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../universal/debtcollection/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkSMTPAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSMTPAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSMTPAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtSMTPUserName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSMTPAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtSMTPPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
